src/python/grapl-plugin-sdk/example_analyzers/process_named_svchost.py

```diff
@@ -0,0 +1,81 @@
+"""
+This is meant as an extremely simple Analyzer to get the pipeline to fire
+during integration tests.
+"""
+from datetime import datetime
+
+from grapl_plugin_sdk.analyzer.analyzer import (
+    Analyzer,
+    AnalyzerContext,
+    AnalyzerServiceConfig,
+    serve_analyzer,
+)
+from grapl_plugin_sdk.analyzer.query_and_views import NodeQuery, NodeView
+from python_proto.api.graph_query.v1beta1.messages import (
+    NodePropertyQuery,
+    StringFilter,
+    StringOperation,
+)
+from python_proto.api.plugin_sdk.analyzers.v1beta1.messages import (
+    AnalyzerName,
+    ExecutionHit,
+)
+from python_proto.common import Timestamp
+from python_proto.grapl.common.v1beta1.messages import NodeType, PropertyName
+
+
+class ProcessNamedSvchost(Analyzer):
```

Comment:
# The majority of this analyzer, including its comments like "???" are just copied from the suspicious_svchost_analyzer. The main difference is the node query is way simpler.